### PR TITLE
Updated CFP deadline

### PR DESCRIPTION
**Pull request by patcon**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -31,11 +31,13 @@ layout: splash
 
 ## Conference Call for Participation
 
+**Update:** Extended one week to Mon, May 7, 2018!
```
Comment (dcwalk) [inline index.md]: Can we have this text say "Monday" as the other page?
Also would just favour "Update: Deadline extended until Monday, May 7, 2018"

Comment (dcwalk) [reply index.md]: Finally, not a deal breaker, but I'm wondering if we can use `<mark>` to accentuate the deadline change?

**Review by dcwalk — COMMENTED** [review]
A request on language update and that we use `<mark>` to highlight the switch if you all are okay with that? I think we should aim to merge this in tomorrow AM

File: index.md (hunk 1 above)
Comment (patcon) [reply index.md]: You mean:

> **Update:** Extended one week to Monday, <mark>May 7, 2018</mark>!

    **Update:** Extended one week to Monday, <mark>May 7, 2018</mark>!

Comment (dcwalk) [reply index.md]: Something like that! Or even the whole line?   
`<mark>**Update:** Deadline extended to Monday, May 7, 2018!</mark>`

**Comment by patcon** [comment]
Ready for review :)

**Comment by dcwalk** [comment]
Awesome! I realized after I could have demo'd to show off. Now I feel really nitpicky 🏷

**Pull request merged**